recycleIT-frontend/src/Components/Main/Main.tsx

```diff
@@ -18,98 +20,135 @@ const Main = () => {
     
     const handleChange = (event: React.SyntheticEvent, newValue: string) => {
         setValue(newValue);
-
       };
       const handleClick = () => {
         ref.current?.scrollIntoView({behavior: 'smooth'});
       };
     
-   const ButtonData =[
-    {
-        title:'PLASTIC',
-        id: 'plastic',
-        value: '1'
-    },
-    {
-        title:'PAPERE',
-        id: 'paper',
-        value: '2'
-    },
-    {
-        title:'GLASS',
-        id: 'glass',
-        value: '3'
-    },
-    {
-        title:'METAL',
-        id: 'metal',
-        value: '4'
-    },
-    {
-        title:'ELECTRONIC',
-        id: 'electronic',
-        value: '5'
-    }
-   ]
+    const ButtonData = [
```

Comment:
allign naming with all other constants, I would also advise to change name to something like WASTE_CATEGORIES so that it's clearer what this mock is for. It can also be moved to data-list-mocks file, similar to FACILITIES_TYPES